data/starNames.txt

```diff
@@ -1,2 +1,5 @@
 22 Lib
 WD 1327-083
+ETA1DOR
+spec:ETA1DOR
+spec:MUCOL
```

Comment:
`nameMappings.txt` and `starNames.txt` seem to duplicate things.  Are the star names just the keys of the name mappings?  I bring up the question because it seems like these files could get out of sync.

Reply:
So, this isn't a great answer, but it is _the_ answer: they're not quite the same thing, and this was only ever supposed to be a temporary "just make it work" solution, but here we are. So: `nameMappings.txt` is a list of object names, which, if seen, should be remapped to the corresponding value (in order that a query for its spectrum succeeds). However, `starNames.txt` is a list of objects which should be considered to be stars, _i.e._ "normal" astronomical emitting sources, as opposed to arc lamps or planetary nebulae etc. Spectractor has a hard-coded heuristic which was "everything which is a star begins with `HD`" and that isn't true, so I added a runtime-editable list of things which should also be processed as if they're stars as well.

Reply:
Now, the real question is: how/where should this be documented, and/or is there a better way of dealing with this?

Reply:
I think, personally, I'd punt on this, merging as-is, and then Jérémy and I can discuss coming up with a better mechanism, and then we properly document that. What do you think?

Reply:
I'm fine with solving this in the future.  I just wasn't sure how they were being used at the moment.

Reply:
Great, thanks.